Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealityControllerMappingProfile.asset

```diff
@@ -2189,7 +2189,7 @@ MonoBehaviour:
         id: 2
         description: Menu
         axisConstraint: 2
-      keyCode: 330
+      keyCode: 332
```

Comment:
There's another mapping profile as well (not sure what the difference in mappings is, if we could squash them into the default):

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/prerelease/2.0.0_stabilization/Assets/MixedRealityToolkit.Examples/Demos/Input/Scenes/InputActions/InputActions.MixedRealityControllerMappingProfile.asset#L2192